src/frontend/src/lib/services/auth.services.ts

```diff
@@ -58,7 +59,7 @@ export const signIn = async (
 	}
 };
 
-export const signOut = (): Promise<void> => logout({});
+export const signOut = (resetUrl = false): Promise<void> => logout({ resetUrl });
```

Comment:
Despite having a single parameter, in this case I would still use an object for paramater, because this makes the code more readable in my opinion. WDYT?

```
await signOut(true)
```

is less expressive than

```
await signOut({resetUrl: true})
```

Reply:
@peterpeterparker I think this is a good idea. `await signOut({resetUrl: true})` is more readable than `await signOut(true)`.